### DISCUSSION TITLE
Disable route request printing on stdout

### DISCUSSION THREAD

**Discussion by nullndr** [remix-run/remix#6792] [Proposals]
I have a route that handles a lot of webhooks, and for each request I see something like the following:

```
...
POST /webhook/foo 200 - - 3.885 ms
POST /webhook/foo 200 - - 5.334 ms
POST /webhook/foo 200 - - 3.133 ms
POST /webhook/foo 200 - - 2.809 ms
POST /webhook/foo 200 - - 3.229 ms
POST /webhook/foo 200 - - 14.591 ms
POST /webhook/foo 200 - - 8.264 ms
POST /webhook/foo 200 - - 4.903 ms
POST /webhook/foo 200 - - 3.798 ms
POST /webhook/foo 200 - - 3.512 ms
POST /webhook/foo 200 - - 3.260 ms
POST /webhook/foo 200 - - 2.582 ms
POST /webhook/foo 200 - - 2.226 ms
POST /webhook/foo 200 - - 4.047 ms
POST /webhook/foo 200 - - 2.871 ms
POST /webhook/foo 200 - - 4.860 ms
POST /webhook/foo 200 - - 3.464 ms
POST /webhook/foo 200 - - 3.465 ms
POST /webhook/foo 200 - - 3.099 ms
POST /webhook/foo 200 - - 3.263 ms
POST /webhook/foo 200 - - 3.397 ms
POST /webhook/foo 200 - - 4.475 ms
...
```

Will it be possible to disable the printing of this? I really do not care much about this log, and I could miss something else more important.

I would like to export a config from my route that explicitly disable this, something like:

```tsx
export const loader = () => {
  return null;
}

export const disableRoutePrint = true;
``` 

When the option is `true`, it will disable also all his children routes, except for those that export the opposite.

The `disableRoutePrint` could also be an object in order to enrich it in the future, so it could be something like:

```tsx
export const loader = () => {
  return null;
}

export const routeConfig = {
  disableRoutePrint: true,
};
```

**Comment by sergiodxa** [comment]
This is configured on the HTTP server, not Remix. If you're using Remix App Server you will have to switch to Express (RAS uses Express anyway), you can use `npx rmx-cli eject-ras`.

If you're using another server adapter check how to configure logs there.

**Reply by nullndr** [reply]
Won't it be possible to just use Remix App Server without these logs?

**Reply by kiliman** [reply]
The issue is that RAS comes pre-configured to log via `morgan`.

https://github.com/remix-run/remix/blob/76327d3b9dcfbb93fd57cd9254a7b66342aba704/packages/remix-serve/index.ts#L25

**Reply by sergiodxa** [reply]
And RAS is not configurable, if you want to configure it you need to go to Express. RAS uses express and is intended for when you don't care about the HTTP server, if you care (and you do because you want to customize the logs) it's better to switch to Express.
